releasenotes/notes/fix-trainable-kernel-tutorial-formatting-c97c24b6bc026aef.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
```

Comment:
As this is just a docs fix, and does not involve any code and will not incur any actual code release, we don't do a reno for these.